resources/views/scenarios/index.blade.php

```diff
@@ -38,12 +39,11 @@ class="fa fa-pencil-square-o" aria-hidden="true"></i> Edit
                                 @endforeach
                                 </tbody>
                             </table>
-                            {{--                    {{ $scenarios->links('pagination::bootstrap-4')  }}--}}
+                                                {{ $scenarios->links('pagination::bootstrap-4')  }}
```

Comment:
misschien mooier om in de AppServiceProvider Paginator::useBootstrapFour(); te zetten?
[https://laravel.com/docs/9.x/pagination#using-bootstrap](https://laravel.com/docs/9.x/pagination#using-bootstrap)